FirebaseSessions/Sources/SessionLocalConfig.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+/// Protoco defining the loval override values for the SDK settings.
+protocol SessionLocalConfigProtocol {
+  /// Session enabled flag value.
+  var sessionEnabled: Bool? { get }
+
+  /// Session timeout flag value.
+  var sessionTimeout: TimeInterval? { get }
+
+  /// Session timeout flag value.
+  var sessionSamplingRate: Double? { get }
+}
+
+/// Class that manages the local overrides configs related to the library.
+class SessionLocalConfig: SessionLocalConfigProtocol {
+  static let PlistKey_sessions_enabled = "firebase_sessions_enabled"
+  static let PlistKey_sessions_timeout = "firebase_sessions_timeout"
+  static let PlistKey_sessions_samplingRate = "firebase_sessions_samplingRate"
```

Comment:
Optional: I usually see Plist Keys in Upper Camel Case (eg. FirebaseSessionsEnabled), but totally up to you if you wanna switch

Reply:
Sounds good. I'm fine with that approach.